sounds/src/stereo/notification/message-new-email.oga

```diff
@@ -0,0 +1 @@
+/home/ian/Documents/Pop/gtk-theme/sounds/src/stereo/notification/message.oga
```

Comment:
If these are symlinks they need to be relative

Reply:
Good catch, thanks

Reply:
Fixed now